packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -45,6 +45,7 @@ class InnerBlocks extends Component {
 			__experimentalBlocks,
 			replaceInnerBlocks,
 			__unstableMarkNextChangeAsNotPersistent,
+			setIsController,
```

Comment:
This reads a bit awkward to me. I think something more analogous to other calls, like `controlInnerBlocks` would read more clearly.

Reply:
That definitely sounds better, thanks!